content/master/operations/watchoperation.md

```diff
@@ -0,0 +1,600 @@
+requirements** in each iteration to signal completion. The function in the
+preceding example always calls `response.require_resources()` regardless of
+whether the service exists. This ensures Crossplane knows when to stop calling
+the function.
+
+Common mistake: only requesting resources when missing breaks the stabilization
+contract and causes timeout errors.
+{{</hint>}}
+
+This pattern allows functions to:
+1. Examine the watched resource (injected automatically)
+2. Dynamically determine what other resources the function needs
+3. Request those resources consistently using `response.require_resources()`
+4. Process all resources when available, or provide status when waiting
+
+## Status and monitoring
+
+WatchOperations provide status information about watching:
+
+```yaml
+status:
+  conditions:
+  - type: Synced
+    status: "True"
+    reason: ReconcileSuccess
+  - type: Watching
+    status: "True"
+    reason: WatchActive
+  watchingResources: 12
+  runningOperationRefs:
+  - name: config-validator-anjda
+  - name: config-validator-f0d92
+```
+
+**Key status fields:**
+- **Conditions**: Standard Crossplane conditions (Synced) and WatchOperation-specific conditions:
+  - **Watching**: `True` when the WatchOperation is actively watching resources, `False` when paused or failed
+- **`watchingResources`**: Number of resources under watch
+- **`runningOperationRefs`**: Running Operations created by this WatchOperation
+
+### Events
+
+WatchOperations emit events for important activities:
+- `EstablishWatched` (Warning) - Watch establishment failures
+- `TerminateWatched` (Warning) - Watch termination failures
+- `GarbageCollectOperations` (Warning) - Operation cleanup failures
+- `CreateOperation` (Warning) - Operation creation failures
+- `ReplaceRunningOperation` (Warning) - Operation replacement failures
+
+<!-- vale write-good.TooWordy = NO -->
+### Monitoring
+<!-- vale write-good.TooWordy = YES -->
+
+<!-- vale write-good.TooWordy = NO -->
+Monitor WatchOperations using:
+<!-- vale write-good.TooWordy = YES -->
+
+```shell
+# Check WatchOperation status
+kubectl get watchoperation my-watchop
+
+# View recent Operations created by the WatchOperation
+kubectl get operations -l crossplane.io/watchoperation=my-watchop
+
+# Check watched resource count
+kubectl describe watchoperation my-watchop
+
+# Check events
+kubectl get events --field-selector involvedObject.name=my-watchop
+```
+
+
+## Best practices
+
+### Resource selection
+
+1. **Use specific label selectors** - Prevent unnecessary Operations with 
+   precise filtering
+1. **Avoid high-churn resources** - Be careful watching frequently changing 
+   resources
+1. **Start small** - Begin with narrow selectors and expand as needed
+
+### Event handling
+
+1. **Implement event filtering** - Check generation, deletion timestamp, 
+   and status conditions
+   to avoid processing irrelevant changes
+<!-- vale write-good.TooWordy = NO -->
```

Comment:
this is causing the ordered list numbering to reset back to 1 on https://deploy-preview-956--crossplane.netlify.app/master/operations/watchoperation/#event-handling

Reply:
Ah dang I thought I got all of these. Fixed now.